content/ember-released-5-3.md

```diff
@@ -0,0 +1,128 @@
+<!-- Block start: If there were no deprecations, remove this block -->
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+<!-- Block end -->
+
+For more details on changes in Ember.js 5.3, please review the [Ember.js 5.3.0 release page](https://github.com/emberjs/ember.js/releases/tag/v5.3.0).
+
+---
+
+## EmberData
+
+EmberData is the official data persistence library for Ember.js applications.
+
+### Changes in EmberData 5.3
+
+#### Bug Fixes
+
+EmberData 5.3 introduced 10 bug fixes. Details are in the
+[EmberData 5.3.0 release page](https://github.com/emberjs/data/releases/tag/v5.3.0).
+
+#### Features
+
+EmberData 5.3 introduced several new features.
+
+This release primarily focused on polish for the `RequestManager` story, introducing builder utils, builders for `REST` `ActiveRecord` and `JSON:API` requests as well as serialization utils for `JSON:API` requests.
+
+As part of shipping serialization utils, we implemented several long-awaited cache features to support them:
+
+- `cache.changedRelationships` retrieves a diff of what has been added or removed from each relationship for a resource
+- `cache.hasChangedRelationships` answers whether any relationships have changed for a resource
+- `cache.rollbackRelationships` will reset all relationships for that resource to their "remote" state, removing any local changes.
+
+#### Deprecations
+
+EmberData 5.3 introduced 5 deprecations.
+
+As part of our ongoing efforts to modernize the library and to support the above changes, we implemented several new deprecations.
+
+- Deprecate legacy import paths (`ember-data/`) except for `ember-data/store`
+- Deprecate non-strict-type (e.g. "users" or "myType" instead of "user" and "my-type")
+- Deprecate non-strict-id (e.g . `1` instead of `"1"`)
+
+And in relationships specifically:
+
+- deprecate fixing non-unique relationship payloads (when the same record is referenced multiple times in a related collection)
+- deprecate remote updates clearing local changes (an upcoming blog post will explain the new behavior here in detail)
+
+For more details on changes in EmberData 5.3, please review the
+[EmberData 5.3.0 release page](https://github.com/emberjs/data/releases/tag/v5.3.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
```

Comment:
in this ember-cli release, pnpm support is no longe behind a feature flag, so folks can use `--pnpm`

Reply:
(what's the proper boilerplate for new features?)

Reply:
@NullVoxPopuli I can add it

Reply:
> (what's the proper boilerplate for new features?)

https://github.com/ember-learn/ember-blog/blob/49eea7a790ff6deba6805e5bbf88db8c4b3cb380/post-templates/emberblog-template.md